openapi/schemas/processes-core/execute.yaml

```diff
@@ -1,5 +1,7 @@
 type: object
 properties:
+  processID:
+    type: string
```

Comment:
To align with other parts that use the URI to allow nested processes reference (Part 3) and POST'ing directly on `/jobs` (Part 4), `process: <uri>` would be preferable.

Reply:
Done.